src/scripts.js

```diff
@@ -131,13 +136,54 @@ function displayRecipe(event) {
   if (card.classList.contains('recipes-container__recipe-card')) {
     hide(homeViewSection);
     hide(allRecipesSection);
+    hide(weeklyRecipesSection);
     show(singleRecipeView);
     renderIndividualRecipe(card.id);
   }
 }
 
+function addToWeeklyRecipes() {
+  const recipe = findRecipeTitle();
+  user.addWeeklyRecipe(recipe);
+  hide(addToWeekButton);
+  show(removeFromWeekButton);
+}
+
+function removeFromWeeklyRecipes() {
+  const recipe = findRecipeTitle();
+  user.removeWeeklyRecipe(recipe);
+  hide(removeFromWeekButton);
+  show(addToWeekButton);
+}
+
+function findRecipeTitle() {
+  const recipeTitle = document.querySelector('.recipe-title').innerText;
+  const recipe = recipeRepository.findRecipesByName(recipeTitle);
+  return recipe;
+}
+
 function displayWeeklyRecipes() {
+  show(weeklyRecipesSection);
+  hide(homeViewSection);
+  hide(allRecipesSection);
+  hide(favoriteRecipesSection); 
+  hide(singleRecipeView);
+  
+  checkWeeklyFavorites();
+}
+
+function checkWeeklyFavorites() {
+  console.log(user)
+  if (!user.weeklyFavorites.length) {
+    show(noWeeklyRecipes);
+    hide(weeklyRecipes);
+  }
 
+  if (user.weeklyFavorites.length) {
```

Comment:
This could technically be an `if/else` statement since they're opposite conditions.